Changelog.rst

```diff
@@ -24,6 +24,8 @@ version 3.16.2
 * Fix bug in `cf.read` when reading UM files that caused LBPROC value
   131072 (Mean over an ensemble of parallel runs) to be ignored
   (https://github.com/NCAS-CMS/cf-python/issues/737)
+* New keyword parameters to `cf.wi`: ``open_lower`` and ``open_upper``
```

Comment:
See `query.py` comments on API

Reply:
Not sure what there is to be done here, given your feedback as registered (on the PR and in person). Please let me know if you had anything specific in mind to tweak or add, here.

Reply:
All good.